docs/source/functional.rst

```diff
@@ -152,6 +151,11 @@ equalizer_biquad
 
 .. autofunction:: equalizer_biquad
 
+filtfilt
+----------------
```

Comment:
Strange thing about RST format, the length of the horizontal line is supposed to be same as the title.

```suggestion
--------
```

Reply:
thanks, fixed